### PR TITLE
Trying to loopify again

### PR DESCRIPTION
**Pull request by Casao**
Tested under bash this time, it works now (the pwd was in the loop instead of outside, doh)

### CODE REVIEW

**Comment by ausmith** [comment]
:+1: looks better.

File: pbp.sh [hunk 1]
```diff
@@ -1,27 +1,20 @@
...
+consumer-session
+domain-values
+consumer-main"
+
+ROOT=$(pwd)
+
+for dir in $REPOS
+do
+  echo "$(tput setaf 5)Pulling and bundling ${dir}...$(tput sgr0)"
+  cd "${ROOT}/${dir}"
```
Comment (oochelz) [inline pbp.sh]: ROOT!

Comment (oochelz) [inline pbp.sh]: $(pwd), rather.

Comment (oochelz) [inline pbp.sh]: Why does this work here and not the way it was before?

Comment (Casao) [inline pbp.sh:14]: `$(pwd)` runs the pwd command in every iteration of the loop. For example, `/Users/cewen/code` is my directory. First time, it gets consumer-layouts and goes into `/Users/cewen/code/consumer-layouts`. Then on the next iteration, consumer-domain, it isn't going up a directory so it becomes `/users/cewen/code/consumer-layouts/consumer-domain`

By just storing the root directory outside, it gets set to `/Users/cewen/code` and goes directly from that every time.

**Pull request merged**